EDMCOverlay/EDMCOverlay/EDMCOverlay.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>EDMCOverlay</RootNamespace>
     <AssemblyName>EDMCOverlay</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Intentional?

Reply:
Nope. I had to convert the project and this might be a side effect. I don't fully understand what this means to be hones ;)

Reply:
It means it's getting compiled for a slightly earlier version of .Net.  maybe you need to install a visual studio service pack?